v2/util/algorithm_identifier.go

```diff
@@ -11,13 +11,13 @@ import (
 	cryptobyte_asn1 "golang.org/x/crypto/cryptobyte/asn1"
 )
 
-// additional OIDs not provided by the x509 package
```

Comment:
I fixed some `godot` nits in this file and then gave up when I saw how many others needed updating. I added `godot` to the linter ignore list for now.